include/ocpn_plugin.h

```diff
@@ -1845,13 +1949,36 @@ extern DECL_EXP CommDriverResult WriteCommDriverN2K(
     const std::shared_ptr<std::vector<uint8_t>> &payload);
 
 /**
- * Special NMEA2000 requirements
- * NMEA2000 bus protocol device management requires that devices writing on
- * the bus must inform all bus listeners of the specific PGNs that may be
- * transmitted by this device. Once configured, this bus management process
- * will be handled transparently by the OCPN core drivers. It is only
- * necessary for plugins wishing to write to the NMEA2000 bus to register the
- * specific PGNs that they anticipate using, with the selected driver.
+ * RegisterTXPGNsRegister PGNs (Parameter Group Numbers) that this application
+ * intends to transmit on an NMEA 2000 network.
+ *
+ * This function is part of the NMEA 2000 "plug and play" capability.
+ * Before any device can transmit messages on an NMEA 2000 network, it must
```

Comment:
@TwoCanPlugIn 

The previous comments were:

> Special NMEA2000 requirements.  NMEA2000 bus protocol device management requires that devices writing on the bus must inform all bus listeners of the specific PGNs that may be transmitted by this device.

Are you saying this is not true? I was trying to improve the code comments by providing the short description (purpose) and then rephrasing the existing comments.